api/api.go

```diff
@@ -78,6 +78,18 @@ func (app api) postHandler(w http.ResponseWriter, r *http.Request) {
 	io.WriteString(w, s)
 }
 
+func (app api) healthHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-type", "application/json")
+
+	if r.Method != http.MethodGet {
+		writeError(w, "Essa URL aceita apenas o método GET.", http.StatusMethodNotAllowed)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+	io.WriteString(w, "{ \"status\": \"OK\" }")
```

Comment:
Precisamos mesmo de conteúdo na mensagem? Apenas o HTTP 200 não basta para o Kubernetes?

Se não precisar, podemos deixar apenas:

```suggestion
	w.WriteHeader(http.StatusOK)
```

Se for preciso manter, que tal escrever a _string_ assim para melhor legibilidade e manutenção?

```
	io.WriteString(w, `{ "status": "OK" }`)
```

Reply:
Ok, basta sim, vou manter só o status então. Devo atualizar amanhã esse pullrequest.